Plugins/Wox.Plugin.Calculator/Languages/en.xaml

```diff
@@ -4,5 +4,7 @@
 
     <system:String x:Key="wox_plugin_caculator_plugin_name">Calculator</system:String>
     <system:String x:Key="wox_plugin_caculator_plugin_description">Allows to do mathematical calculations.(Try 5*3-2 in Wox)</system:String>
-    
+    <system:String x:Key="wox_plugin_calculator_not_a_number">Not a Number (NaN)</system:String>
+    <system:String x:Key="wox_plugin_calculator_expression_not_complete">Expression wrong or incomplete (Did you forget some Brackets?)</system:String>
```

Comment:
I think you mean `Parentheses ()`  rather than `Brackets []`?

Reply:
Jup, sure thanks for looking over it !